src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -1226,6 +1252,11 @@ void doIndirectImport(ImportTask task) throws IOException {
                                             featureType.getQualifiedName(), FeatureTypeInfo.class);
                     if (resource != null) {
                         calculateBounds(resource);
+                        if (task.getUpdateMode() == UpdateMode.REPLACE) {
+                            FeatureType schema = resource.getFeatureType();
+
+                            Collection<PropertyDescriptor> attributes = schema.getDescriptors();
+                        }
```

Comment:
Looks like this is debugging code and can be remove, changed to log schema details